dali/python/nvidia/dali/ops.py

```diff
@@ -63,10 +63,10 @@ def __floordiv__(self, other):
     def __rfloordiv__(self, other):
         return _arithm_op("div", other, self)
 
-    # def __neg__(self):
-    #     return _arithm_op("minus", self)
-    # def __pos__(self):
-    #     return _arithm_op("plus", self)
+    def __neg__(self):
+        return _arithm_op("minus", self)
+    def __pos__(self):
+        return _arithm_op("plus", self)
```

Comment:
If unary plus is a no-op (which may depend on type?), then please just return self.

Reply:
Done